src/OSPSuite.SimModel/Simulation.cs

```diff
@@ -654,12 +654,9 @@ protected virtual void Dispose(bool disposing)
          if (_disposed)
             return;
 
-         if (disposing)
-         {
-            SimulationImports.DisposeParameterInfoVector(_allParameters);
-            SimulationImports.DisposeSpeciesInfoVector(_allSpecies);
-            SimulationImports.DisposeSimulation(_simulation);
-         }
+         SimulationImports.DisposeParameterInfoVector(_allParameters);
+         SimulationImports.DisposeSpeciesInfoVector(_allSpecies);
+         SimulationImports.DisposeSimulation(_simulation);
```

Comment:
That was wrong. _Unmanaged_ resources must be released **always**, not only `if disposing`.
s. also https://docs.microsoft.com/en-us/dotnet/standard/garbage-collection/implementing-dispose

Reply:
indeed. I don't think i ever used this variable in code, Interesting